highlight_view.py

```diff
@@ -361,6 +364,14 @@ def get_icon_scope(icon, error):
         return "region.whitish"  # hopefully a white color
 
 
+def undraw(view):
+    current_region_keys = get_regions_keys(view)
+    current_linter_keys = {key for key in current_region_keys
+                           if key.startswith('SL.')}
+    for key in current_linter_keys:
+        view.erase_regions(key)
+
```

Comment:
All regions start with 'SL.' so you don't need to filter here. So, basically it is ok to do `for key in current_region_keys`. 

But afterwards you must call `remember_region_keys(view, set())` to clean the store!

Reply:
Excellent. Thanks!